tests/utils_test_2025_12_10_lidov3.py

```diff
@@ -0,0 +1,896 @@
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[2],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=REGISTER_GROUPS_IN_OPERATOR_GRID_FACTORY,
+                    permissions=create_permissions(interface.OperatorGrid(OPERATOR_GRID), "registerGroup") + create_permissions(interface.OperatorGrid(OPERATOR_GRID), "registerTiers")[2:]
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[3],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=REGISTER_TIERS_IN_OPERATOR_GRID_FACTORY,
+                    permissions=create_permissions(interface.OperatorGrid(OPERATOR_GRID), "registerTiers")
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[4],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=UPDATE_GROUPS_SHARE_LIMIT_IN_OPERATOR_GRID_FACTORY,
+                    permissions=create_permissions(interface.OperatorGrid(OPERATOR_GRID), "updateGroupShareLimit")
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[5],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=SET_JAIL_STATUS_IN_OPERATOR_GRID_FACTORY,
+                    permissions=create_permissions(interface.IVaultsAdapter(VAULTS_ADAPTER), "setVaultJailStatus")
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[6],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=UPDATE_VAULTS_FEES_IN_OPERATOR_GRID_FACTORY,
+                    permissions=create_permissions(interface.IVaultsAdapter(VAULTS_ADAPTER), "updateVaultFees")
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[7],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=FORCE_VALIDATOR_EXITS_IN_VAULT_HUB_FACTORY,
+                    permissions=create_permissions(interface.IVaultsAdapter(VAULTS_ADAPTER), "forceValidatorExit")
+                ),
+                emitted_by=easy_track,
+            )
+
+            validate_evmscript_factory_added_event(
+                event=vote_events[8],
+                p=EVMScriptFactoryAdded(
+                    factory_addr=SOCIALIZE_BAD_DEBT_IN_VAULT_HUB_FACTORY,
+                    permissions=create_permissions(interface.IVaultsAdapter(VAULTS_ADAPTER), "socializeBadDebt")
+                ),
+                emitted_by=easy_track,
+            )
+
```

Comment:
Consider adding check, that after the Aragon vote has passed, creation of the vaults via VaultsFactory still reverts